# Translations and Tipping the Balance {-}

**Led by Matt Moscou and Peter Van Esse**

*Exploiting Knowledge of Plant Pathogen Interactions for Durable Disease Resistance*

Lorem ipsum dolor sit amet, consectetur adipiscing elit. In iaculis sagittis metus quis malesuada. Vestibulum laoreet vel tortor at tempor. Mauris blandit volutpat risus, et placerat odio varius eget. Cras vitae diam sollicitudin justo vehicula ultrices. Curabitur consequat ornare odio ac hendrerit. Mauris bibendum diam nec gravida molestie. Nullam interdum, nulla eget sollicitudin elementum, ante lorem auctor odio, sit amet varius diam magna et lectus. Sed vehicula velit velit, convallis mollis erat laoreet nec. Cras semper blandit felis vel iaculis. Sed in mollis nulla. Nulla sed egestas odio, nec finibus lectus. In vel porttitor lacus, nec tincidunt neque. Suspendisse at magna non neque congue fermentum at ut augue. Vivamus suscipit finibus tortor, ut accumsan dui pretium nec. Integer luctus eros non convallis ornare.

Pellentesque dapibus arcu rutrum neque tempor, ac consequat nisi consectetur. Nunc eu tortor sed lorem vehicula finibus. Sed sed lacus at erat fringilla auctor. Aenean malesuada mi est, sit amet molestie ante malesuada in. Aenean elit risus, imperdiet quis pretium ac, imperdiet a turpis. Praesent cursus, orci a viverra tincidunt, augue tortor volutpat erat, a elementum nisi ante vel nisl. Vestibulum purus nisl, imperdiet egestas ultricies nec, tempor vitae ligula. Integer dapibus magna vitae orci pharetra rhoncus. Vestibulum in mauris feugiat, pulvinar massa ac, convallis lacus. In hac habitasse platea dictumst.

Aenean sed leo sit amet dolor congue semper. Curabitur non malesuada erat. Phasellus scelerisque rutrum tristique. Duis posuere quam tellus, eu posuere lorem convallis facilisis. Duis euismod tincidunt neque, eu pellentesque nisi mollis sed. Suspendisse eu nunc rutrum massa iaculis rhoncus rutrum nec velit. Nulla et erat quis urna malesuada auctor. Vestibulum sit amet diam maximus, luctus nulla mollis, vestibulum nisi.

```{r maint, echo=FALSE, fig.cap="This image should really be one that nicely summarises your topic. Not Roger.", dpi=96 }
knitr::include_graphics("assets/RPF-thumbnail.jpeg")
```

## Keynote Lecture {-}

### Beat Keller - Molecular analysis of wheat – fungal pathosystems and applications in resistance breeding {-}
**Department of Plant and Microbial Biology, University of Zürich, Switzerland**

Several hundred resistance genes against fungal diseases have been genetically described in the gene pool of wheat and a few of them have been cloned. The molecular analysis of their origin, diversity and function has resulted in insight on their evolution and also suggested better ways for their use in classical as well as molecular breeding. For example, the discovery of molecular suppressor activities of certain powdery mildew resistance genes has resulted in a better understanding of earlier observations in wheat breeding and allows to predict breeding outcomes. The current focus of our work is on two different aspects of disease resistance in wheat: first we study the molecular basis of durable disease resistance and we focus here on the _Lr34_ gene which was originally described as a QTL for durable, quantitative disease resistance against the fungal pathogen leaf rust. Second, we are analyzing at the molecular level the interaction of wheat and the fungal pathogen powdery mildew. This work also includes the identification of pathogen determinants that are involved in resistance. 
The new developments in wheat genomics including the availability of a high-quality reference genome sequence allow us to develop more efficient ways to isolate resistance genes. There is a rapidly increasing number of innovative new approaches to clone genes from the wheat genome which I will discuss briefly, and we can expect many more resistance genes being isolated in the near future. The consequences of these developments for a better use of genetic diversity in wheat resistance breeding will be discussed.
The molecular diversity that has been revealed by studies on agronomically important resistance genes can inspire a number of research directions to improve resistance breeding. For example, natural diversity provides important clues how to engineer immune receptors for broader recognition spectrum. Furthermore, modification of gene expression as well as combination of genes using transgenic approaches has revealed novel ways to improve disease resistance. Several of these approaches will be presented for the case of leaf rust and powdery mildew resistance in wheat.


### About Beat Keller {-}

> Dr. Keller received his PhD from the University of Basel, Switzerland, in 1985, and then was postdoctoral fellow at the Salk Institute for Biological Studies in La Jolla, San Diego with a long-term fellowship of the European Molecular Biology Organization. After returning to Europe he started a research group in collaboration with the wheat breeding program in Switzerland and became a full professor for Plant Molecular Biology at the University of Zurich in 1997.
The group of Dr. Keller has focused on the molecular understanding of fungal disease resistance in the wheat, maize and barley crop plants. This has resulted in the molecular identification of a number of agronomically important resistance genes. More recently, the group has started a large project on the wheat powdery mildew pathogen to understand resistance at the molecular level. In addition, fungal pathogenomics is used to study the evolution of this highly host-specific pathogen.
Dr. Keller was Vice-president of the Swiss Academy of Sciences from 2000-2006, has led several large research consortia in Switzerland and was an ERC Advanced Investigator grant holder from 2010 to 2015. He is a member of the Research Council of the Swiss National Science Foundation and currently Head of the Division of Biology at the University of Zurich.


## Practical Session - Practical Session Title {-}

**Led by Matt Moscou**

### Aims and Objectives {-}

1. Understand different methods for resistance loci
2. Introduce basic R packages for QTL analysis
3. Understand how to link genotype (genetic maps) with phenotype (continuous/discrete measurements) with packages.

Mendelian inheritance of resistance to plant pathogens was first established by Sir Roland Henry Biffen in 1907. By the end of the 20th century, the link of phenotype and genotype was established by map-based cloning of plant resistance genes to a range of plant pathogens. Although substantial advances have been made in cloning resistance genes from model species with small genomes, technical hurdles still exist for large complex genomes that may have little or no sequence information. This practical will consist of a general introduction of the different methodologies and approaches for identifying loci contributing to resistance in complex uncharacterized genomes and a tutorial on using R/qtl to link genotype (genetic maps) with phenotype (both quantitative and qualitative data).

