<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>The Sainsbury Laboratory Summer School 2017</title>
  <meta name="description" content="The handbook for the TSL Summer School 2017">
  <meta name="generator" content="bookdown 0.4 and GitBook 2.6.7">

  <meta property="og:title" content="The Sainsbury Laboratory Summer School 2017" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="The handbook for the TSL Summer School 2017" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="The Sainsbury Laboratory Summer School 2017" />
  
  <meta name="twitter:description" content="The handbook for the TSL Summer School 2017" />
  




  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html">
<link rel="next" href="surface-immunity.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">TSL Summer School 2017</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome to The Sainsbury Laboratory Summer School 2017</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html"><i class="fa fa-check"></i>Course schedule</a><ul>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#monday-31st-july"><i class="fa fa-check"></i>Monday 31st July</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#tuesday-1st-august"><i class="fa fa-check"></i>Tuesday 1st August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#wednesday-2nd-august"><i class="fa fa-check"></i>Wednesday 2nd August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#thursday-3rd-august"><i class="fa fa-check"></i>Thursday 3rd August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#friday-4th-august"><i class="fa fa-check"></i>Friday 4th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#saturday-5th-august"><i class="fa fa-check"></i>Saturday 5th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#sunday-6th-august"><i class="fa fa-check"></i>Sunday 6th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#monday-7th-august"><i class="fa fa-check"></i>Monday 7th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#tuesday-8th-august"><i class="fa fa-check"></i>Tuesday 8th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#wednesday-9th-august"><i class="fa fa-check"></i>Wednesday 9th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#thursday-10th-august"><i class="fa fa-check"></i>Thursday 10th August</a></li>
<li class="chapter" data-level="" data-path="course-schedule.html"><a href="course-schedule.html#friday-11th-august"><i class="fa fa-check"></i>Friday 11th August</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html"><i class="fa fa-check"></i>Resistance Proteins</a><ul>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html#keynote-lecture"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html#jijie-chai---structural-study-of-plant-receptor-kinases"><i class="fa fa-check"></i>Jijie Chai - Structural Study of Plant Receptor Kinases</a></li>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html#about-jijie-chai"><i class="fa fa-check"></i>About Jijie Chai</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html#practical-session---model-pathosystems-and-effector-triggered-immunity-readouts"><i class="fa fa-check"></i>Practical Session - Model pathosystems and effector triggered immunity readouts</a><ul>
<li class="chapter" data-level="" data-path="resistance-proteins.html"><a href="resistance-proteins.html#aims-and-objectives"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><i class="fa fa-check"></i>Genomic Resources and Bioinformatics for Plant Microbe Interactions.</a><ul>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html#keynote-lecture-1"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html#diane-g.o.-saunders---developing-new-tools-for-interrogating-cereal-invaders"><i class="fa fa-check"></i>Diane G.O. Saunders - Developing new tools for interrogating cereal invaders</a></li>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html#about-diane-saunders"><i class="fa fa-check"></i>About Diane Saunders</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html#practical-session---from-sequence-data-to-candidate-gene"><i class="fa fa-check"></i>Practical Session - From Sequence Data to Candidate Gene</a><ul>
<li class="chapter" data-level="" data-path="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html"><a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html#aims-and-objectives-1"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="effectors-and-immunity.html"><a href="effectors-and-immunity.html"><i class="fa fa-check"></i>Effectors and Immunity</a><ul>
<li class="chapter" data-level="" data-path="effectors-and-immunity.html"><a href="effectors-and-immunity.html#keynote-lecture-2"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li><a href="effectors-and-immunity.html#jens-boch---a-battle-for-life-and-death---evolution-of-tales-in-plant-pathogenic-xanthomonas-bacteria">Jens Boch - A battle for life and death - Evolution of TALEs in plant-pathogenic <em>Xanthomonas</em> bacteria</a></li>
<li class="chapter" data-level="" data-path="effectors-and-immunity.html"><a href="effectors-and-immunity.html#about-jens-boch"><i class="fa fa-check"></i>About Jens Boch</a></li>
</ul></li>
<li><a href="effectors-and-immunity.html#practical-session---studying-effector-function-in-planta">Practical Session - Studying Effector Function <em>in planta</em></a><ul>
<li class="chapter" data-level="" data-path="effectors-and-immunity.html"><a href="effectors-and-immunity.html#aims-and-objectives-2"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html"><i class="fa fa-check"></i>Surface Immunity</a><ul>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html#keynote-lecture-3"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html#stefanie-ranf---specificity-of-lore-dependent-lipopolysaccharide-immune-sensing-in-arabidopsis"><i class="fa fa-check"></i>Stefanie Ranf - Specificity of LORE-dependent lipopolysaccharide immune sensing in Arabidopsis</a></li>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html#about-stefanie-ranf"><i class="fa fa-check"></i>About Stefanie Ranf</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html#practical-session---analysing-surface-immunity"><i class="fa fa-check"></i>Practical Session - Analysing Surface Immunity</a><ul>
<li class="chapter" data-level="" data-path="surface-immunity.html"><a href="surface-immunity.html#aims-and-objectives-3"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html"><i class="fa fa-check"></i>Cellular Defence</a><ul>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html#keynote-lecture-4"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html#paul-birch---the-delivery-and-activity-of-late-blight-effector-proteins-that-suppress-plant-immunity"><i class="fa fa-check"></i>Paul Birch - The Delivery and Activity of late blight effector proteins that suppress plant immunity</a></li>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html#about-paul-birch"><i class="fa fa-check"></i>About Paul Birch</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html#practical-session---live-cell-imaging-and-investigation-of-subcellular-membrane-trafficking"><i class="fa fa-check"></i>Practical Session - Live Cell Imaging and Investigation of Subcellular Membrane Trafficking</a><ul>
<li class="chapter" data-level="" data-path="cellular-defence.html"><a href="cellular-defence.html#aims-and-objectives-4"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html"><i class="fa fa-check"></i>Fuctional Plant Genomics</a><ul>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html#keynote-lecture-5"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html#daniel-croll---retracing-genome-evolution-of-pathogens-during-rapid-disease-emergence-in-agricultural-ecosystems"><i class="fa fa-check"></i>Daniel Croll - Retracing genome evolution of pathogens during rapid disease emergence in agricultural ecosystems</a></li>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html#about-daniel-croll"><i class="fa fa-check"></i>About Daniel Croll</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html#practical-session---functional-plant-genomics"><i class="fa fa-check"></i>Practical Session - Functional Plant Genomics</a><ul>
<li class="chapter" data-level="" data-path="fuctional-plant-genomics.html"><a href="fuctional-plant-genomics.html#aims-and-objectives-5"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><i class="fa fa-check"></i>Application of Discovery and Targeted Proteomics in Plant Pathogen Interactions</a><ul>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html#keynote-lecture-6"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html#delphine-pflieger---quantitative-phosphoproteomic-analysis-reveals-shared-and-specific-targets-of-arabidopsis-mapkinases-mpk3-mpk4-and-mpk6"><i class="fa fa-check"></i>Delphine Pflieger - Quantitative phosphoproteomic analysis reveals shared and specific targets of Arabidopsis MAPkinases MPK3, MPK4 and MPK6</a></li>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html#about-delphine-pflieger"><i class="fa fa-check"></i>About Delphine Pflieger</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html#practical-session---analysing-and-interpreting-mass-spec-data"><i class="fa fa-check"></i>Practical Session - Analysing and interpreting mass spec data</a><ul>
<li class="chapter" data-level="" data-path="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html"><a href="application-of-discovery-and-targeted-proteomics-in-plant-pathogen-interactions.html#aims-and-objectives-6"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html"><i class="fa fa-check"></i>Translations and Tipping the Balance</a><ul>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html#keynote-lecture-7"><i class="fa fa-check"></i>Keynote Lecture</a><ul>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html#beat-keller---molecular-analysis-of-wheat-fungal-pathosystems-and-applications-in-resistance-breeding"><i class="fa fa-check"></i>Beat Keller - Molecular analysis of wheat – fungal pathosystems and applications in resistance breeding</a></li>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html#about-beat-keller"><i class="fa fa-check"></i>About Beat Keller</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html#practical-session---practical-session-title"><i class="fa fa-check"></i>Practical Session - Practical Session Title</a><ul>
<li class="chapter" data-level="" data-path="translations-and-tipping-the-balance.html"><a href="translations-and-tipping-the-balance.html#aims-and-objectives-7"><i class="fa fa-check"></i>Aims and Objectives</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html"><i class="fa fa-check"></i>General Information</a><ul>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#arriving-into-norwich"><i class="fa fa-check"></i>Arriving into Norwich</a><ul>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#arriving-by-air"><i class="fa fa-check"></i>Arriving by Air</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#arriving-by-train"><i class="fa fa-check"></i>Arriving by Train</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#arriving-by-coach"><i class="fa fa-check"></i>Arriving by Coach</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#arriving-by-car-and-parking"><i class="fa fa-check"></i>Arriving by Car and Parking</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#getting-from-and-into-central-norwich-by-bus"><i class="fa fa-check"></i>Getting from and into Central Norwich by Bus</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#checking-into-accomodation"><i class="fa fa-check"></i>Checking into Accomodation</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#meals"><i class="fa fa-check"></i>Meals</a><ul>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#breakfast-and-lunch"><i class="fa fa-check"></i>Breakfast and Lunch</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#evening-meals"><i class="fa fa-check"></i>Evening Meals</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#getting-around"><i class="fa fa-check"></i>Getting Around</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#site-registration"><i class="fa fa-check"></i>Site Registration</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#emergency-contacts"><i class="fa fa-check"></i>Emergency Contacts</a><ul>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#internal-emergency-first-aid"><i class="fa fa-check"></i>Internal Emergency First Aid</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#hospital"><i class="fa fa-check"></i>Hospital</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#transport"><i class="fa fa-check"></i>Transport</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#norwich-and-norwich-research-park"><i class="fa fa-check"></i>Norwich and Norwich Research Park</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#wifi-connections"><i class="fa fa-check"></i>Wifi Connections</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#social-media"><i class="fa fa-check"></i>Social Media</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#map"><i class="fa fa-check"></i>Map</a></li>
<li class="chapter" data-level="" data-path="general-information.html"><a href="general-information.html#excursion"><i class="fa fa-check"></i>Excursion</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">The Sainsbury Laboratory Summer School 2017</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="effectors-and-immunity" class="section level1 unnumbered">
<h1>Effectors and Immunity</h1>
<p><strong>Led by Sophien Kamoun</strong></p>
<p>One of nature’s many secrets in biology is how filamentous plant pathogens cause disease on their hosts. We are making tremendous progress towards understanding this process largely due to breakthroughs in “effector biology” <span class="citation">(Hogenhout et al. <a href="#ref-Hogenhout:2009em">2009</a>; Win et al. <a href="#ref-Win:2012jd">2012</a>)</span>. Effectors are proteins secreted by pathogens to suppress host immune systems and manipulate plant physiology to enable pathogen colonization (Figure <a href="effectors-and-immunity.html#fig:maineff">5</a>A).</p>

<div class="figure"><span id="fig:maineff"></span>
<img src="assets/eff.png" alt="The concept of effectors in plant immunity. Infectious pathogens such as bacteria, fungi, oomycete, and nematodes deliver effectors at the interface of the host plant (apoplastic effectors, AE) or inside the cell (cytoplasmic effectors, CE). Host-translocated (cytoplasmic) effectors are delivered into the host cytoplasm through a type-III secretion pilus or specialized infectious structures called haustoria that form within the cell. Pathogen effectors traffic to various compartments, bind, and manipulate different host proteins called targets. Depending on their localization in the cells, these targets are designated as apoplastic effector target (AET) and cytoplasmic effector target (CET). Effector–target interactions impact the outcome of the interaction between the pathogen and its host. In susceptible genotypes (A), these molecular interactions can alter plant cell processes and suppress immune responses, leading to effector-triggered susceptibility (ETS) and host colonization. In resistant genotypes (B), these interactions are perceived by key sensing receptors of the immune system that, in turn, stop pathogen growth. Cell surface pattern recognition receptors (PRRs) detect pathogen-associated molecular patterns (PAMPs), apoplastic effectors, and/or apoplastic effector–target interactions to initiate PRRtriggered immunity (PTI). Intracellular nucleotide-binding receptors (NB-LRR) induce NB-LRR-triggered immunity (ETI) on recognition of cytoplasmic effectors and/or cytoplasmic effectors–target interactions (Reproduced from Win et al. (2012))." width="228" />
<p class="caption">
Figure 5: <strong><em>The concept of effectors in plant immunity</em></strong>. Infectious pathogens such as bacteria, fungi, oomycete, and nematodes deliver effectors at the interface of the host plant (apoplastic effectors, AE) or inside the cell (cytoplasmic effectors, CE). Host-translocated (cytoplasmic) effectors are delivered into the host cytoplasm through a type-III secretion pilus or specialized infectious structures called haustoria that form within the cell. Pathogen effectors traffic to various compartments, bind, and manipulate different host proteins called targets. Depending on their localization in the cells, these targets are designated as apoplastic effector target (AET) and cytoplasmic effector target (CET). Effector–target interactions impact the outcome of the interaction between the pathogen and its host. In susceptible genotypes (A), these molecular interactions can alter plant cell processes and suppress immune responses, leading to effector-triggered susceptibility (ETS) and host colonization. In resistant genotypes (B), these interactions are perceived by key sensing receptors of the immune system that, in turn, stop pathogen growth. Cell surface pattern recognition receptors (PRRs) detect pathogen-associated molecular patterns (PAMPs), apoplastic effectors, and/or apoplastic effector–target interactions to initiate PRRtriggered immunity (PTI). Intracellular nucleotide-binding receptors (NB-LRR) induce NB-LRR-triggered immunity (ETI) on recognition of cytoplasmic effectors and/or cytoplasmic effectors–target interactions (Reproduced from <span class="citation">Win et al. (<a href="#ref-Win:2012jd">2012</a>)</span>).
</p>
</div>
<p>We can predict an “effectorome”, or effector repertoire, of a pathogen from its genome sequence based on several criteria that we know about effectors <span class="citation">(Saunders et al. <a href="#ref-Saunders:2012bw">2012</a>)</span>. These criteria include, presence of a secretory signal peptide, lack of transmembrane domains, lack of mitochondrial targeting signals, presence of known/conserved motifs such as RXLR, upregulation of their transcripts during infection, specific localization in host cells, similarity to known effectors of other plant pathogen species, etc. However, one persistent question is: What are the intrinsic functions of these effectors? To find out, we express tagged effectors in plants and observe their effect on plant morphology and physiology [such as hypersensitive response (HR) cell death] using a technique called the agrobacteria-mediated transient transformation procedure <span class="citation">(Kaipla et al. <a href="#ref-Kapila:1997">1997</a>)</span>, also known as agroinfiltration. In addition, to work out what these effectors might be doing in plants, we derive clues from the effector-associated plant proteins that we identify by performing co-immunoprecipitation (co-IP) of tagged effectors followed by mass spectrometry (MS) <span class="citation">(Win, Kamoun, and Jones <a href="#ref-Win:2011dw">2011</a>)</span>. These plant proteins constitute potential targets of the effectors. We confirm the association between the effectors and the potential targets in an independent system known as yeast-two-hybrid interaction assays <span class="citation">(Fields and Song <a href="#ref-Fields:1989dm">1989</a>)</span>, or in vitro interaction assays. Once we know the effector targets, we investigate them to find out if they play important roles in plant-pathogen interactions by knocking down their transcripts in plants by virus-induced gene silencing (VIGS) <span class="citation">(Lindbo et al. <a href="#ref-Lindbo:1993kp">1993</a>)</span> and observing the effects on plant responses to infection.</p>
<p></p>
<div id="keynote-lecture-2" class="section level2 unnumbered">
<h2>Keynote Lecture</h2>
<div id="jens-boch---a-battle-for-life-and-death---evolution-of-tales-in-plant-pathogenic-xanthomonas-bacteria" class="section level3 unnumbered">
<h3>Jens Boch - A battle for life and death - Evolution of TALEs in plant-pathogenic <em>Xanthomonas</em> bacteria</h3>
<p><strong>Leibniz Universität Hannover, Institute of Plant Genetics, 30419 Hannover, Germany</strong></p>
<p>Plant pathogens embark upon a fierce battle with their plant hosts over tissue colonization and nutrient acquisition. This has deadly consequences for both partners of the interaction, but also for us if crop plants are involved. Plant pathogenic <em>Xanthomonas oryzae</em> bacteria can cause substantial losses in rice, making this pathogen a key threat to our food production. Like many other plant pathogens, <em>Xanthomonas</em> bacteria translocate a collection of effector proteins into eukaryotic host cells to function as potent weapons. Among these effectors, TALEs (<u>t</u>ranscription <u>a</u>ctivator-<u>l</u>ike <u>e</u>ffectors) play a pivotal role. TALEs act as transcription factors within the plant cell to induce expression of specific target genes and support the bacterial infection. The DNA-binding mechanism of TALEs is unique and based on a number of characteristic 34-amino acid tandem repeats. Each repeat recognizes one base in the target DNA via one precisely positioned amino acid of a repeat-variable diresidue (RVD). This modular architecture allows the rearrangement of TALE repeats to evolve new DNA-binding specificities matching new virulence targets. Whole genome sequences of several <em>Xanthomonas oryzae</em> strains recently revealed both, the diversity and conservation of TALEs in <em>X. oryzae</em>. In combination with plant transcript analyses and TALE target prediction we now have an outstanding opportunity to identify TALE virulence targets and decipher the pathogen’s strategy. The constant evolutionary battle between pathogen and host has caused evasive mutations in plant promoters that preclude TALE-binding. Matching modifications in TALEs lead to the development of aberrant repeats with novel DNA-binding behaviours. This exemplifies the power of the molecular arms race between pathogen and host. The simple way of generating any desired DNA-binding specificity has initiated a worldwide use of TALEs for biotechnological applications. Designer TALEs and TALE nucleases (TALEN) have successfully been used for highly efficient genome reprogramming and genome editing. Today, crops and livestock with novel traits as well as powerful tools for molecular medicine have already been developed using TALEs.</p>
<p>Further reading:</p>
<ol style="list-style-type: decimal">
<li><span class="citation">Boch et al. (<a href="#ref-Boch:2009fc">2009</a>)</span> Breaking the code of DNA binding specificity of TAL-type III effectors.</li>
<li><span class="citation">Richter et al. (<a href="#ref-Richter:2014kz">2014</a>)</span> A TAL effector repeat architecture for frame shift binding.</li>
<li><span class="citation">Grau et al. (<a href="#ref-Grau:2016ft">2016</a>)</span> AnnoTALE: bioinformatics tools for identification, annotation, and nomenclature of TALEs from <em>Xanthomonas</em> genomic sequences</li>
</ol>
</div>
<div id="about-jens-boch" class="section level3 unnumbered">
<h3>About Jens Boch</h3>
<blockquote>
<p>Professor for Plant Biotechnology (2015-present), Leibniz Universität Hannover, Germany Assistant/Associate Professor (1999-2015), Martin Luther University Halle-Wittenberg, Germany Postdoctoral Fellow (1997-1999), Washington University in St. Louis, USA Postdoctoral Fellow (1996-1997), MPI for Terrestrial Microbiology, Marburg, Germany Ph.D. (1996), Philipps University Marburg, Germany</p>
</blockquote>
</div>
</div>
<div id="practical-session---studying-effector-function-in-planta" class="section level2 unnumbered">
<h2>Practical Session - Studying Effector Function <em>in planta</em></h2>
<p><strong>Led by Joe Win</strong></p>
<p>We are interested in how plants defend themselves against pathogens, and how the pathogens suppress these defences. Plant immunity is governed by recognition of effectors by plant immune receptors (Figure <a href="effectors-and-immunity.html#fig:maineff">5</a>B). Once recognized, several signalling processes are initiated by the receptors and defence is mounted against the invading pathogen. We have identified a novel network of plant immune components that is involved in defence signalling of Solanaceous plants <span class="citation">(Wu et al. <a href="#ref-Wu:2016bio">2016</a>)</span>. We study their function and activities by deploying the methods such as co-IP, HR assays and VIGS as described above.</p>
<p>In these practical sessions, we will guide you through the techniques and methods described above to probe plant-microbe interactions with a focus on effectors and plant immunity.</p>
<div id="aims-and-objectives-2" class="section level3 unnumbered">
<h3>Aims and Objectives</h3>
<ol style="list-style-type: decimal">
<li>To be familiar with current knowledge of effectors and their roles in infection of host plants.</li>
<li>To understand the roles of helper components in plant immune signalling network.</li>
<li>To understand the theory and practice of methods and techniques that are routinely used to study effectors and plant immunity</li>
</ol>
<p>In this practical session, you will be invited to choose one of the following techniques for hands-on experience guided by the instructors.</p>
<ol style="list-style-type: decimal">
<li>In planta co-immunoprecipitation of effectors and associated plant proteins, and analysis of effector-plant protein interactions <em>in vitro</em></li>
<li>Yeast two-hybrid assay to study protein-protein interaction</li>
<li>Agroinfiltration and virus-induced gene silencing to study gene functions in plants</li>
</ol>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-Hogenhout:2009em">
<p>Hogenhout, Saskia A, Renier A L Van der Hoorn, Ryohei Terauchi, and Sophien Kamoun. 2009. “Emerging concepts in effector biology of plant-associated organisms.” <em>Molecular Plant-Microbe Interactions : MPMI</em> 22 (2): 115–22.</p>
</div>
<div id="ref-Win:2012jd">
<p>Win, J, A Chaparro-Garcia, K Belhaj, D G O Saunders, K Yoshida, S Dong, S Schornack, et al. 2012. “Effector biology of plant-associated organisms: concepts and perspectives.” <em>Cold Spring Harbor Symposia on Quantitative Biology</em> 77: 235–47.</p>
</div>
<div id="ref-Saunders:2012bw">
<p>Saunders, Diane G O, Joe Win, Liliana M Cano, Les J Szabo, Sophien Kamoun, and Sylvain Raffaele. 2012. “Using hierarchical clustering of secreted protein families to classify and rank candidate effectors of rust fungi.” <em>PloS One</em> 7 (1): e29847.</p>
</div>
<div id="ref-Kapila:1997">
<p>Kaipla, J, R de Rycke, M van Montagu, and G Angenon. 1997. “An Agrobacterium-mediated transient gene expression system for intact leaves.” <em>Plant Science</em> 122 (1): 101–8.</p>
</div>
<div id="ref-Win:2011dw">
<p>Win, Joe, Sophien Kamoun, and Alexandra M E Jones. 2011. “Purification of effector-target protein complexes via transient expression in Nicotiana benthamiana.” <em>Methods in Molecular Biology (Clifton, N.J.)</em> 712: 181–94.</p>
</div>
<div id="ref-Fields:1989dm">
<p>Fields, S, and O Song. 1989. “A novel genetic system to detect protein-protein interactions.” <em>Nature</em> 340 (6230): 245–46.</p>
</div>
<div id="ref-Lindbo:1993kp">
<p>Lindbo, J A, L Silva-Rosales, W M Proebsting, and W G Dougherty. 1993. “Induction of a Highly Specific Antiviral State in Transgenic Plants: Implications for Regulation of Gene Expression and Virus Resistance.” <em>The Plant Cell</em> 5 (12): 1749–59.</p>
</div>
<div id="ref-Boch:2009fc">
<p>Boch, Jens, Heidi Scholze, Sebastian Schornack, Angelika Landgraf, Simone Hahn, Sabine Kay, Thomas Lahaye, Anja Nickstadt, and Ulla Bonas. 2009. “Breaking the code of DNA binding specificity of TAL-type III effectors.” <em>Science (New York, N.Y.)</em> 326 (5959): 1509–12.</p>
</div>
<div id="ref-Richter:2014kz">
<p>Richter, Annekatrin, Jana Streubel, Christina Blücher, Boris Szurek, Maik Reschke, Jan Grau, and Jens Boch. 2014. “A TAL effector repeat architecture for frameshift binding.” <em>Nature Communications</em> 5 (March): 3447.</p>
</div>
<div id="ref-Grau:2016ft">
<p>Grau, Jan, Maik Reschke, Annett Erkes, Jana Streubel, Richard D Morgan, Geoffrey G Wilson, Ralf Koebnik, and Jens Boch. 2016. “AnnoTALE: bioinformatics tools for identification, annotation, and nomenclature of TALEs from Xanthomonas genomic sequences.” <em>Scientific Reports</em> 6 (February): 21077.</p>
</div>
<div id="ref-Wu:2016bio">
<p>Wu, C H, A Abd-El-Haliem, T O Bozkurt, K Belhaj, R Terauchi, J H Vossen, and S Kamoun. 2016. “NLR signaling network mediates immunity to diverse plant pathogens.” <em>BioRxiv</em>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="genomic-resources-and-bioinformatics-for-plant-microbe-interactions-.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="surface-immunity.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/04-effectors.Rmd",
"text": "Edit"
},
"download": ["summer_school_2017.pdf", "summer_school_2017.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
